pkg/api/orchestrators.go

```diff
@@ -80,15 +79,15 @@ func GetOrchestratorVersionProfileListVLabs(orchestrator, version string, window
 	return orchList, nil
 }
 
-// GetOrchestratorVersionProfileListV20170930 returns v20170930 OrchestratorVersionProfileList object per (optionally) specified orchestrator and version
-func GetOrchestratorVersionProfileListV20170930(orchestrator, version string) (*v20170930.OrchestratorVersionProfileList, error) {
+// GetOrchestratorVersionProfileList returns unversioned OrchestratorVersionProfileList object per (optionally) specified orchestrator and version
+func GetOrchestratorVersionProfileList(orchestrator, version string) (*OrchestratorVersionProfileList, error) {
```

Comment:
there is already an existing `getOrchestratorVersionProfileList` function. How does this one compare to it?

Reply:
Good question, the public function is just wrapping the private function. The reason for this is that OrchestratorVersionProfileList has required metadata like ID and Name that AKS needs. If having two functions with the same name is confusing, we could rename the public function to be GetExtendedOrchestratorVersionProfileList. Also, this function could be housed in AKS RP instead of AKS-engine - I am going to double check that it can be moved and then move it over to avoid bloating aks-engine.